ibtk/src/utilities/libmesh_utilities.cpp

```diff
@@ -40,6 +40,10 @@
 #include "libmesh/numeric_vector.h"
 #include "libmesh/system.h"
 
```

Comment:
```suggestion
```

Reply:
Let's try to keep the `#include` blocks grouped together, so that `clang-format` will keep them in order.

Reply:
Got it. Thanks!